core/src/main/scala/org/apache/spark/scheduler/LiveListenerBus.scala

```diff
@@ -87,7 +87,9 @@ private[spark] class LiveListenerBus(conf: SparkConf) {
    * of each other (each one uses a separate thread for delivering events), allowing slower
    * listeners to be somewhat isolated from others.
    */
-  private def addToQueue(listener: SparkListenerInterface, queue: String): Unit = synchronized {
+  private[spark] def addToQueue(
```

Comment:
Is it necessary to make this change?

Reply:
Change it in order to call it in StreamingQueryListenerBus.